neurolib/optimize/evolution/evolutionaryUtils.py

```diff
@@ -134,18 +122,17 @@ def plotPopulation(
 
     # Gather all the fitnesses in one list and print the stats
     # validPop = [p for p in pop if not np.isnan(p.fitness.score)]
-    validPop = [p for p in pop if not np.any(np.isnan(p.fitness.values))]
+    # validPop = [p for p in pop if not np.any(np.isnan(p.fitness.values))]
+    validPop = [p for p in pop if not (np.isnan(p.fitness.values).any() or np.isinf(p.fitness.values).any()) ]
```

Comment:
I think I've seen this checking whether a population is valid, what about a short convenience function?
```python
def _is_pop_valid(pop):
    return not (np.isnan(pop.fitness.values).any() or np.isinf(pop.fitness.values).any())
```
then just
```python
validPop = [p for p in pop if _is_pop_valid(p)]
```
and also in other places where you check validness of the population